kubeless-rbac.jsonnet

```diff
@@ -24,7 +24,7 @@ local controller_roles = [
   {
     apiGroups: ["kubeless.io"],
     resources: ["functions", "kafkatriggers", "httptriggers", "cronjobtriggers"],
-    verbs: ["get", "list", "watch", "update"],
+    verbs: ["create", "get", "list", "watch", "update", "patch"],
```

Comment:
I see we use patch (which is essentially equivalent to update) for managing finalizers.  When does the controller serviceaccount need to *create* these resources?

Reply:
the resources in this rbac rule (functions and triggers)

Reply:
Got it. You are right. `create` should not be there. Will fix it.